components/CompositionCollectionView/src/ExpressionsFork/Expressions/ExpressionNodes/ColorNode.cs

```diff
@@ -0,0 +1,137 @@
+#nullable disable
```

Comment:
License should still be top thing, does this tie into CS8602? Ideally we should just be good had writing nullable safe code, but probably easier to just do at project level in csproj and track via a todo later.

Reply:
Nullability is worth doing a pass on before merging in here. This PR will already be nontrivial to bring up to date with our latest tooling.